pages.fr/common/zsh.md

```diff
@@ -14,12 +14,16 @@
 
 - Exécute un script :
 
-`zsh {{chemin/vers/le/script.zsh}}`
+`zsh {{chemin/vers/script.zsh}}`
 
 - Exécute un script en affichant chaque commande avant de l'exécuter :
 
-`zsh --xtrace {{path/to/script.zsh}}`
+`zsh --xtrace {{chemin/vers/script.zsh}}`
 
 - Démarre une session shell interactive en mode verbeux, qui affiche chaque commande avant de l'exécuter :
 
 `zsh --verbose`
+
+- Exécute une commande spécifique dans `zsh` sans motifs génériques d'expansion des noms de fichier :
+
+`noglob {{command}}`
```

Comment:
```suggestion
`noglob "{{command}}"`
```

Optional fix ([original page](https://github.com/tldr-pages/tldr/blob/main/pages/common/zsh.md) can be updated): add missing quotes.